index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <a href="#welcome-section">About</a>
+      <a href="#projects">Work</a>
+      <a href="#contact">Contact</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+      <h2 class="projects_title">These are some of my projects</h2>
+      <div class="projects-container">
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+        <div class="project-tile">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"/>
+          <p>Map data accross the globe</p>
+          <a href="github.com" target="_blank">View  Project</a>
+        </div>
+
+      </div>
+      <a href="github.com" target="_blank" id="profile-link">Show All</a>
+    </section>
+
+    <section id="contact">
+      <h1>Let's work together...</h1>
+      <p>How do you take your coffee?</p>
+    </section>
+    <section id="footer">
+      <a id="profile-link" href="#" target="_blank">Facebook</a>
+      <a id="profile-link" href="#" target="_blank">Github</a>
+      <a id="profile-link" href="#" target="_blank">Twitter</a>
+      <a id="profile-link" href="#" target="_blank">Send an Email</a>
+      <a id="profile-link" href="#" target="_blank">Call me</a>
+    </section>
```

Comment:
I think the social list should be a <ul> with every item bing an <li>. This's better use of semantic tags.

Reply:
Please be sure to be as clear as possible when writing your requirements. Sadly, I did not understand what you are asking for 😥 Also, don't forget to explain the reason behind your changes 🤓

Reply:
Edit:
I think the social links represent a list therefore they should be contained in a semantic <ul> tag while every single link should be inside a <li> tag. By applying semantic tags in the correct context, you make it easier for humans to understand your code, search engines to index your website, and even make your website more accessible by making it easy for screen readers to read it.